src/framework/preferences/js/Enactors.js

```diff
@@ -459,7 +459,11 @@ https://github.com/fluid-project/infusion/raw/main/Infusion-LICENSE.txt
                 createOnEvent: "onCreateTOCReady",
                 options: {
                     listeners: {
-                        "afterRender.boilAfterTocRender": "{fluid.prefs.enactor.tableOfContents}.events.afterTocRender"
+                        "afterRender.boilAfterTocRender": "{fluid.prefs.enactor.tableOfContents}.events.afterTocRender",
+                        "onCreate.show": {
```

Comment:
Either this is a bug in the ToC core implementation in which case this fix should go there, or else it is part of the enactor's workflow, in which case it should manifest as an extra call to {enactor}.applyToc - either way we should have a comment here explaining the situation and reference to FLUID-6696